sktime/classification/interval_based/_drcif.py

```diff
@@ -193,26 +195,26 @@ def __init__(
         self._att_subsample_size = att_subsample_size
         self._min_interval = min_interval
         self._max_interval = max_interval
-        self._base_estimator = base_estimator
 
         super(DrCIF, self).__init__()
 
+        if isinstance(base_estimator, str):
+            if base_estimator.lower() == "dtc":
+                self._base_estimator = DecisionTreeClassifier(criterion="entropy")
+            elif base_estimator.lower() == "cit":
+                self._base_estimator = ContinuousIntervalTree()
+        elif isinstance(base_estimator, BaseEstimator):
+            self._base_estimator = copy.deepcopy(base_estimator)
```

Comment:
using `_clone_estimator` `sktime.base._base` should be safer here

Reply:
I changed here